core/src/main/java/com/linecorp/armeria/common/Cookie.java

```diff
@@ -68,11 +68,24 @@ static Cookie of(String name, String value) {
      *
      * @param name the name of the {@link Cookie}
      * @param value the value of the {@link Cookie}
+     *
+     * @deprecated Use {@link #secureBuilder(String, String)} instead to create secure cookie.
      */
+    @Deprecated
     static CookieBuilder builder(String name, String value) {
         return new CookieBuilder(name, value);
     }
 
+    /**
+     * Returns a newly created {@link CookieBuilder} which builds a {@link Cookie} with secure settings.
+     *
+     * @param name the name of the {@link Cookie}
+     * @param value the value of the {@link Cookie}
+     */
+    static CookieBuilder secureBuilder(String name, String value) {
+        return new CookieBuilder(name, value).secure(true).httpOnly(true);
```

Comment:
`"SameSite=Strict"` could be considered as the sensible default for secure cookies.

Reply:
That's right. I will set `Strict` into `SameSite` attribute and update javadoc comment to notice that users can't send cookie for cross-site requests.